### ISSUE TITLE
[BUG] With updated version of vagrant/vagrant-libvirt, MAC address reported on Vagranfile is in wrong format.

### ISSUE THREAD

**Issue by ssasso** [ipspace/netlab#815]
```
An error occurred while executing the action on the 'r1'
machine. Please handle this error then try again:

There are errors in the configuration of this machine. Please fix
the following errors and try again:

Libvirt Provider:
* Configured NIC MAC '08-4F-A9-00-00-01' is not in 'xx:xx:xx:xx:xx:xx' or 'xxxxxxxxxxxx' format
```

@ipspace can you please verify with your current installed version if the MAC format with only hex digits (no `-` or `:`) still works fine?
In that case I will create a patch to always put only `[0-9A-Fa-f]` of the mac address

**Comment by ipspace** [comment]
vagrant-libvirt version 0.11.2 (the version installed with the "official" install script) handles MAC addresses without delimiters, so we can go for `[0-9A-F]{12}` format. We should also document this as a breaking change in the release notes, as it might break some ancient deployments.

**Comment by ipspace** [comment]
This behavior seems to be caused by https://github.com/vagrant-libvirt/vagrant-libvirt/commit/1741ee2f6d462bba6b635ca8e3cb8103db459ae6 in 0.12.1.  It looks like vagrant-libvirt's canonical format is aa:aa:aa:aa:aa:aa so we should go with that one.

**Comment by ipspace** [comment]
Implemented in 2409966f8f53ede6fa6de8ad40aa918988f48268 using `mac_unix_expanded` format (as that seems to be the canonical format used by vagrant-libvirt plugin)
